src/librustc/diagnostics.rs

```diff
@@ -1444,6 +1444,51 @@ lint name). Ensure the attribute is of this form:
 ```
 "##,
 
+E0453: r##"
+A lint check attribute was overruled by a `forbid` directive set as an
+attribute on an enclosing scope, or on the command line with the `-F` option.
+
+Example of erroneous code:
+
+```compile_fail
+#![forbid(non_snake_case)]
+
+#[allow(non_snake_case)]
+fn main() {
+    let MyNumber = 2; // error: allow(non_snake_case) overruled by outer
+                      // forbid(non_snake_case)
+}
+```
+
+The `forbid` lint setting makes code that fails the lint check result in a
```

Comment:
This is a bit hard to read. Could you rewrite it please? For example:

"The `forbid` lint setting transform the corresponding compilation warning into a hard error. It also prevents itself from being overridden by inner attributes."